scripts/preload_models.py

```diff
@@ -36,6 +40,19 @@
 transformer = CLIPTextModel.from_pretrained(version)
 print('...success')
 
+
+print('preloading Stable Diffusion model file...', end='')
+# sd1_4 = Path(hf_hub_download("CompVis/stable-diffusion-v-1-4-original", "sd-v1-4.ckpt"))
+# inpainting = Path(hf_hub_download("runwayml/stable-diffusion-inpainting", "sd-v1-5-inpainting.ckpt"))
+sd1_5 = Path(hf_hub_download("runwayml/stable-diffusion-v1-5", "v1-5-pruned.ckpt"))
+
+model_dest = Path('models/ldm/stable-diffusion-v1/model.ckpt')
+model_dest.parent.mkdir(parents=True, exist_ok=True)
+sd1_5.resolve(strict=True).link_to(model_dest)
```

Comment:
I did this because it was the smallest change from the existing process. But an alternative would be to rely on `huggingface_hub`'s cache mechanism directly wherever this model is loaded, instead of managing this `models/` directory.

Reply:
This is great, but don't you still need to establish an account on Hugging Face and agree to the license terms? Where does that happen?

Reply:
Yep, you need to click that button on the model page. After that, you can use the API to download instead of the browser.

Reply:
But what is the initial UX? Does the script cause a browser to pop up with appropriate page loaded? Then, on subsequent runs, are we reading a token or other auth from somewhere? I'm really not clear on how this works.

Reply:
The UX is good only ***if*** you follow the directions I've added to INSTALL_LINUX: use `huggingface-cli login` and agreeing to the model terms _before_ running `preload_models`.

It's definitely worth checking out what happens if you do things in any other order. You probably get a faceful of traceback with some "make sure you're logged in and have access" messages mixed in.

On subsequent runs, huggingface_hub retrieves the token from wherever `huggingface-cli login` stored it.